Cargo.toml

```diff
@@ -24,6 +24,11 @@ bitflags = { version = "1.0", default-features = false }
 defmt = { version = "0.2.0", optional = true }
 rand_core = { version = "0.6.3", optional = true, default-features = false }
 
+ccm = "0.4.4"
```

Comment:
crypto usually causes giant code size bloat, it should be optional.

It should be excluded If 802154 is not used at least. In this case I think the linker will remove the bloat anyway, so maybe it's just a code size optimiation.

I don't know how often is unencrypted 802154 used in the real world? If the answer is "never" we could force "if you use 802154 you have crypto", but otherwise it should be possible to build 802154 support without crypto.

Reply:
I also have no idea how often unencrypted is used in the real world.
I put it behind a feature flag (another one ;p).